README.md

```diff
@@ -53,31 +55,56 @@ video = pipe("<my-awesome-prompt>").frames[0]
+Create two files: one containing line-separated prompts, and the other containing line-separated paths to video files (
+video file paths must be relative to the path you specify with `--data_root`). Let's understand this better with an
+example!
 
-Assume you've specified `--data_root` as `/dataset`, and that this directory contains the files: `prompt.txt` and `videos.txt`.
+Assume your specified `--data_root` is `/dataset`, and this directory contains the files `prompts.txt` and `videos.txt`.
 
-The `prompt.txt` file should contain line-separated prompts:
+### Preparing the Prompts Dataset
+
+The `prompts.txt` file should contain line-separated prompts. Please note that the prompts must be in English, and it is
+recommended to use the [prompt polishing script](https://github.com/THUDM/CogVideo/blob/main/inference/convert_demo.py)
+for refinement. Alternatively, you can use [CogVideo-caption](https://huggingface.co/THUDM/cogvlm2-llama3-caption) for
+captioning:
 
 ```
 A black and white animated sequence featuring a rabbit, named Rabbity Ribfried, and an anthropomorphic goat in a musical, playful environment, showcasing their evolving interaction.
-A black and white animated sequence on a ship's deck features a bulldog character, named Bully Bulldoger, showcasing exaggerated facial expressions and body language. The character progresses from confident to focused, then to strained and distressed, displaying a range of emotions as it navigates challenges. The ship's interior remains static in the background, with minimalistic details such as a bell and open door. The character's dynamic movements and changing expressions drive the narrative, with no camera movement to distract from its evolving reactions and physical gestures.
+A black and white animated sequence on a ship’s deck features a bulldog character, named Bully Bulldoger, showcasing exaggerated facial expressions and body language. The character progresses from confident to focused, then to strained and distressed, displaying a range of emotions as it navigates challenges. The ship’s interior remains static in the background, with minimalistic details such as a bell and open door. The character’s dynamic movements and changing expressions drive the narrative, with no camera movement to distract from its evolving reactions and physical gestures.
 ...
+
 ```
 
-The `videos.txt` file should contain line-separate paths to video files. Note that the path should be _relative_ to the `--data_root` directory.
+### Preparing the Video Dataset
+
+The framework supports the following specific resolutions and frame counts:
+
+- **Resolutions (Width * Height) Supported**:
+    - Any width * height divisible by 32 between 256 and 2048. For example, `720 * 480`, `1920 * 1020`.
+
+- **Frame Counts (Frames) Supported**:
+    - 16, 24, 32, 48, 64, 80. The frame count must be divisible by 4.
```

Comment:
I think better to say `4 * k` or `4 * k + 1`

Reply:
We should then also denote what is "k". I think the current proposed change is good enough.